02-microarray/00-intro-to-microarray.Rmd

```diff
@@ -6,3 +6,60 @@ output:
     toc: true
     toc_float: true
 ---
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Introduction to Microarray technology](#introduction-to-microarray-technology)
+  - [Microarray data **strengths**:](#microarray-data-strengths)
+  - [Microarray data **limitations**:](#microarray-data-limitations)
+- [About quantile normalization](#about-quantile-normalization)
+- [More resources on microarray technology:](#more-resources-on-microarray-technology)
+  - [Why doesn't the gene I care about show up in this microarray dataset?](#why-doesnt-the-gene-i-care-about-show-up-in-this-microarray-dataset)
+- [References](#references)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction to Microarray technology
+
+Data analyses are generally not "one size fits all"; this is particularly true between RNA-seq vs microarray data.
+This tutorial has example analyses [organized by technology](../01-getting-started/getting-started.html#about-how-this-tutorial-book-is-structured) so you can follow examples that are more closely tailored to the nature of the data at hand. 
+
+As with all experimental methods, microarray has strengths and limitations that you should consider in regards to your scientific questions. 
+
+### Microarray data **strengths**:  
+
+- Less expensive than RNA-seq - can afford more replicates and get higher statistical power [@Tarca2006]. 
+- Microarray has a faster turn-around than RNA-seq [@LCSciences2014].
+
+### Microarray data **limitations**:  
+
+- If a transcript doesn't have a probe designed to it, you won't get it measured; can't be used for transcript discovery [@Mantione2014]. 
+- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
+- As is true for all techniques that involve nucleotide hybiridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition (like GC bias). 
+
+To help address these probe nucleotide composition biases, refine.bio uses [SCAN](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3508193/pdf/nihms401888.pdf) normalization methods prior to your downloads [@Piccolo2012].
```

Comment:
Does the SCAN comment go only with the third bullet point? Maybe don't have the space between them

Reply:
It's at least a spin-off of the third bullet point for sure. But it doesn't exaclty qualify to be in our list of "limitations". I'll think about how to possibly restructure.

Reply:
(Maybe as a parenthetical statement with the bullet point.)

Reply:
Hmm... I don't like a super long parenthetical statement.

Reply:
I'm okay with the spacing, but if there are more complaints I will change it. (We have the same [kind of structure on RNA-seq section header](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#rna-seq-data-limitationsbiases)